iotdb/src/main/java/org/apache/iotdb/db/service/CloseMergeService.java

```diff
@@ -122,7 +122,7 @@ public void start() throws StartupException {
           .format("Failed to start %s because of %s", this.getID().getName(),
               e.getMessage());
       LOGGER.error(errorMessage);
-      throw new StartupException(errorMessage);
+      throw new StartupException(errorMessage, e);
```

Comment:
Why here throw an exception while an error log is printed.

Reply:
Removed.